tests/test_packages/test_skills_integration/test_http_echo.py

```diff
@@ -40,6 +40,10 @@ def test_echo(self):
         self.add_item("connection", "fetchai/http_server:0.16.0")
         self.add_item("skill", "fetchai/http_echo:0.14.0")
         self.set_config("agent.default_connection", "fetchai/http_server:0.16.0")
+        self.set_config(
```

Comment:
JFYI
found the case when multiple skills are ok to get http server message.
skills can handle various routes/server names, so can filter messages by some criteria.
anyway it's a specific casse